frontend/src/resizer.pug

```diff
@@ -0,0 +1,20 @@
+#app-wrapper(
```

Comment:
Is there a customary practice to name files that are interlinked to each other (see eg below)? If so, let's change the file name to `resize.pug` to be consistent with the other file names.

E.g.:
`authorship.pug` <--> `v-authorship.js` <--> `v-authorship.scss`

Reply:
updated, thanks for the review!

could you click the update branch button below as well? ( have ensured its a clean merge )
Travis dosen't return the status otherwise due to permission issues